### PR TITLE
feat: add compatibility with `RxJs` and `Most`

### PR DESCRIPTION
**Pull request by Fyzu**
The `Property` must provide minimal [proposal](https://github.com/tc39/proposal-observable#observablefrom) ES observable compliance as: it must have a `[Symbol.observable]()` method that returns an object with a well-behaved `.subscribe()` method.

`RxJs` and` Most` support this behavior:
- [RxJs](https://github.com/ReactiveX/rxjs/blob/master/src/internal/types.ts#L107)
- [Most](https://github.com/cujojs/most/blob/bfed148f1bfd8bc06a972a2cb17599371d8da52f/docs/api.md#mostfrom)

### CODE REVIEW

File: packages/core/src/property.ts [hunk 1]
```diff
@@ -3,9 +3,11 @@ import { mergeMany, newEmitter } from './emitter'
 import { newAtom } from './atom'
 import { Env, Time } from './clock'
 import { memoMany } from '@frp-ts/utils'
+import { InteropObservable, newInteropObservable, observableSymbol } from './interop-observable'
 
 export interface Property<A> extends Observable<Time> {
 	readonly get: () => A
+	[Symbol.observable]: () => InteropObservable<A>
```
Comment (raveclassic) [inline packages/core/src/property.ts:10]: Can we make this `readonly`?

File: packages/core/src/index.ts [hunk 2]
```diff
@@ -4,6 +4,7 @@ export * as emitter from './emitter'
 export { AddEventListenerOptions, EventListenerOptions, EventTarget, Emitter } from './emitter'
 export * as observable from './observable'
 export { Observer, Subscription, Observable } from './observable'
+export * as interopObservable from './interop-observable'
```
Comment (raveclassic) [inline packages/core/src/index.ts:7]: Why do you want to export this from the package root? This is an internal implementation detail

Comment (Fyzu) [reply packages/core/src/index.ts:7]: Needed for `fp-ts` library. How we should handle it?

File: packages/fp-ts/src/property.ts [hunk 3]
```diff
@@ -24,16 +24,32 @@ export const instance: Applicative1<URI> = {
 	URI: URI,
 	map: (fa, f) => {
 		const memoF = memo1(f)
+		const get = () => memoF(fa.get())
 		return {
-			get: () => memoF(fa.get()),
+			get,
 			subscribe: fa.subscribe,
+			[interopObservable.observableSymbol]: () => interopObservable.newInteropObservable(fa.subscribe, get),
```
Comment (raveclassic) [inline packages/fp-ts/src/property.ts:31]: Ah, I see.
Ok, how about introducing a constructor for `Property`? 
```typescript
export const newProperty = <Value>(get: () => Value, subscribe: Observable<Time>['subscribe']): Property<Value> => ({
  get,
  subscribe,
  [observableSymbol]: () => newInteropObservable(subscribe, get)
})
```

File: packages/core/src/index.ts (hunk 2 above)
Comment (Fyzu) [reply packages/core/src/index.ts:7]: Should import from `core/src/interop-observable`?

**Review by raveclassic — APPROVED** [review]

**Pull request merged**